s3backgrounddelete/ut/cortx_s3_iem_tests.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from s3backgrounddelete.IEMutil import IEMutil
+
+def test_send():
```

Comment:
@nileshgovande  to run this test code, we need configure-iem.sh script which i have created as part of init.sh 

If we dont want to integrate this script then we should remove the UTs

Reply:
Check for the existance of config file and if present, continue

Reply:
done